docs/sources/review/lint-prose/index.md

```diff
@@ -59,3 +59,68 @@ The task title contains a verb and an object. For example: "Create a dashboard".
 
 <!-- vale Grafana.Quotes = YES -->
 ```
+
+## Use Vale in VS Code
+
+You can use Vale to lint your current document in VS Code.
+
+1. Clone the [Technical Documentation](https://github.com/grafana/technical-documentation) repository.
+
+   ```bash
+   git clone git@github.com:grafana/technical-documentation.git
+   ```
+
+1. Download [Vale](https://github.com/errata-ai/vale/releases).
+
+   {{% admonition type="note" %}}
+   Refer to the Linux installation steps at [GitHub Releases](https://vale.sh/docs/vale-cli/installation/#github-releases). Verify that you are downloading the latest build of Vale for Linux.
+   {{% /admonition %}}
+
+   {{< code >}}
+
+   ```linux
+   wget https://github.com/errata-ai/vale/releases/download/v2.28.0/vale_2.28.0_Linux_64-bit.tar.gz
+   mkdir bin && tar -xvzf vale_2.28.0_Linux_64-bit.tar.gz -C bin
+   export PATH=./bin:"$PATH"
+   ```
+
+   ```macos
+   brew install vale
+   ```
+
+   {{< /code >}}
+
+1. Create a `vale.ini` file in your home directory or in a working directory with the following contents:
+
+   ```bash
+   StylesPath = /FULL_PATH_TO_REPO/technical-documentation/linters/vale
+   MinAlertLevel = suggestion
+
+   [*.md]
+   BasedOnStyles = Grafana
+   ```
+
+   Replace `FULL_PATH_TO_REPO` with the full path to the cloned Technical Documentation repository. For example, in Linux you could set StylesPath to `/home/username/git-repos/technical-documentation/linters/vale` and in macOS, you could set it to `/Users/username/git-repos/technical-documentation/linters/vale`. The path depends on where you cloned the git repository.
+
+1. Install the [Vale VS Code extension](https://marketplace.visualstudio.com/items?itemName=chrischinchilla.vale-vscode) in VS Code.
+
+   1. Start VS Code.
+   1. Press Ctrl+P, paste the following command, and press Enter.
+
+   ```
+   ext install ChrisChinchilla.vale-vscode
+   ```
+
+1. Configure the Vale VS Code extension.
+
+   1. Press Ctrl+Shift+X and select the Vale VS Code extension.
+   1. Select the gear icon.
+   1. Set `Vale › Vale CLI: Config` to the path to your `vale.ini` file. For example, on Linux that could be `/home/USERNAME/vale.ini` and on macOS, that could be `/Users/USERNAME/vale.ini`. The path depends on where you created the `vale.ini` file.
+   1. Set `Vale › Vale CLI: Path` to the path for the vale executable. For example, in Linux, that could be `/home/USERNAME/bin/vale` and on macOS, that could be `/usr/local/bin/vale`.
```

Comment:
How does one determine this just in case this isn't correct?

Reply:
It really depends on how you install it - the challenge of a single stand-alone executable. 

In Linux, you can download the zip file.. and put it anywhere you want/ The steps for Linux suggest putting it in `/home/USERNAME/bin`. 

In macOS, it also depends... if you use brew, it'll probably be OK to link it `/usr/local/bin/vale` since that's supposed to be a default location on macOS... but it depends on local brew config.  I'm not sure if using `which vale` will report the path to the executable... this works on Linux.